### DISCUSSION TITLE
Disable "Always show full URLs" not working

### DISCUSSION THREAD

**Discussion by achtchaern** [Eloston/ungoogled-chromium#2163] [General]
Hi! Is there a way to NOT show the complete URLs? I mean, the option is there (in the context menu of the Omnibox), but it's not working.

**Comment by PF4Public** [comment]
It could be this patch: https://github.com/ungoogled-software/ungoogled-chromium/blob/master/patches/extra/ungoogled-chromium/disable-formatting-in-omnibox.patch
